src/main/java/dev/agones/AgonesSDK.java

```diff
@@ -64,7 +61,7 @@ public void startHealthTask(int period, TimeUnit unit) {
     }
 
     public void health() {
-        getEndpoints(AgonesEndpoints.class).health();
+        getEndpoints(AgonesEndpoints.class).health(new Empty());
```

Comment:
I'm thinking this would be better as a singular static instance

Reply:
Agreed. Potentially `Empty#empty` (or something similar?) returning the static instance.

Reply:
@theminecoder Not sure what you mean here. Do you mean something like @MylesDev pointed out?

Reply:
Yeh something like that should suffice. The key point is we don't need to make a new `Empty` instance for each request

Reply:
Not sure how to structure this. Does this look right? Shouldn't it be initialized at least once somewhere?
```java
public class Empty {
    public static Empty empty;

    public Empty() {}
}
```

Reply:
`public class Empty {
    
    private static final Empty INSTANCE = new Empty();
    
    private Empty() {}
    
    public static Empty empty() {
        return INSTANCE;
    }
}`

Reply:
> ```java
> public class Empty {
>     
>     private static final Empty INSTANCE = new Empty();
>     
>     public static Empty empty() {
>         return INSTANCE;
>     }
> 
>     private Empty() {}
> }
> ```

Thanks for that, seems very logical indeed. I had something like that already but was worried about my IDE complaining about a utility class instantiation.

I pushed the changes